src/components/ClusterManager.js

```diff
@@ -330,6 +332,29 @@ export class NewClusterModal extends PureComponent {
   }
 }
 
+const errorNotified = { current: false }
```

Comment:
nit: This could be a React `ref` or an `atom` instead of a special thing. Also, it might be simpler to store the ID of the last reported errored cluster, and then just compare against that, which would eliminate the need to reset the flag in several places.

We should also consider how this behaves across several projects. Right now, it would suppress notifications after the first one.